kin/account.py

```diff
@@ -0,0 +1,347 @@
+from .config import MIN_ACCOUNT_BALANCE, SDK_USER_AGENT
+from .blockchain.utils import is_valid_secret_key, is_valid_address
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class KinAccount:
+    """Account class to perform authenticated actions on the blockchain"""
+    def __init__(self, seed, client, channels, channel_secret_keys, create_channels):
+        # Set the internal sdk
+        self._client = client
+
+        # Verify seed
+        if not is_valid_secret_key(seed):
+            raise ValueError('invalid secret key: {}'.format(seed))
+
+        # Set keypair
+        self.keypair = Keypair(seed)
+        # check that sdk wallet account exists and is activated, issuer should continue without activation
+        if self._client.get_account_status(self.keypair.public_address) != AccountStatus.ACTIVATED and \
+                        self.keypair.public_address != self._client.kin_asset.issuer:
+            raise KinErrors.AccountNotActivatedError
+
+        if channels is not None and channel_secret_keys is not None:
+            raise ValueError("Account cannot be initialized with both 'channels'"
+                             " and 'channel_secret_keys' parameters")
+
+        if channel_secret_keys is not None:
+            # Use given channels
+            for channel_key in channel_secret_keys:
+                # Verify channel seed
+                if not is_valid_secret_key(channel_key):
+                    raise ValueError('invalid channel key: {}'.format(channel_key))
+                # Check that channel accounts exists (they do not have to be activated).
+                channel_address = Keypair.address_from_seed(channel_key)
+                if self._client.get_account_data(channel_address) == AccountStatus.NOT_CREATED:
+                    raise KinErrors.AccountNotFoundError
+            self.channel_secret_keys = channel_secret_keys
+
+        elif channels is not None:
+            # Generate the channels for the user
+            self.channel_secret_keys = [Keypair.generate_hd_seed(seed, str(channel)) for channel in range(channels)]
+        else:
+            # Use the base account as the only channel
+            self.channel_secret_keys = [seed]
+
+        if create_channels:
+            if channels is None:
+                raise ValueError("create_channels can only be used with the channels parameter")
+
+            # Create the channels using the base account
+            if self.channel_secret_keys == [seed]:
+                raise ValueError('There are no channels to create')
+            base_account = KinAccount(seed,self._client, None, None, False)
+
+            # Verify that there is enough XLM to create the channels
+            # Balance should be at least (Number of channels + yourself) * (Minimum account balance + fees)
+            if (len(self.channel_secret_keys) + 1) * (MIN_ACCOUNT_BALANCE + 0.00001) > \
```

Comment:
const (0.00001)

Reply:
I will add it as a base fee in the config file